examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py

```diff
@@ -270,8 +270,9 @@ def get_user_model():
         torchscript = True
     if args.woq_algo == "AutoRound" and is_habana_framework_installed():
         print("Quantizing model with AutoRound on HPU")
-        check_torch_compile_with_hpu_backend()
-        set_envs_for_torch_compile_with_hpu_backend()
+        if args.quantize:
```

Comment:
Please attach the example test report for reference.

Reply:
opt_125m_woq_autoround_int4: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/151/
opt_125m_woq_autoround_int4_hpu: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/inc_gaudi_test/152/

Reply:
Both extension tests are running on the HPU. Is that expected?

Reply:
Yes, here I intend to compare the impact of special env variables for autoround.